src/In.ProjectEKA.HipService/Program.cs

```diff
@@ -47,5 +64,14 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args)
                 .UseSerilog()
                 .UseStartup<Startup>();
         }
+        
+        private static ElasticsearchSinkOptions ConfigureElasticSink(IConfigurationRoot configuration, string environment)
+        {
+            return new ElasticsearchSinkOptions(new Uri(configuration["ElasticConfiguration:Uri"]))
+            {
+                AutoRegisterTemplate = true,
+                IndexFormat = $"{Assembly.GetExecutingAssembly().GetName().Name.ToLower().Replace(".", "-")}-{environment?.ToLower().Replace(".", "-")}-{DateTime.UtcNow:yyyy-MM}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5420628)